mayastor/tests/block_device_nvmf.rs

```diff
@@ -1194,8 +1281,14 @@ async fn wipe_device_blocks(is_unmap: bool) {
     const BUF_SIZE: u64 = 32768;
     const OP_OFFSET: u64 = 12 * 1024 * 1024;
 
+    static DEVICE_NAME: OnceCell<String> = OnceCell::new();
+
     // Read completion callback.
```

Comment:
FYI: I know you didn't change this line but the comment doesn't seem to match the callback function. I think it is a copy paste error that occurs a few times in this file.